samples/charts/category-chart/trendline/Pages/CategoryChartTrendline.razor

```diff
@@ -0,0 +1,142 @@
+﻿@page "/CategoryChartTrendline"
+
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <span className="igOptions-label">Trend Line Type: </span>
+        <select @onchange="OnChartTrendlineChanged">
+            <option>LinearFit</option>
+            <option>QuadraticFit</option>
+            <option>CubicFit</option>
+            <option>QuarticFit</option>
+            <option>QuinticFit</option>
+            <option>LogarithmicFit</option>
+            <option>ExponentialFit</option>
+            <option>PowerLawFit</option>
+            <option>SimpleAverage</option>
+            <option>ExponentialAverage</option>
+            <option>ModifiedAverage</option>
+            <option>CumulativeAverage</option>
+            <option>WeightedAverage</option>
+            <option>None</option>
```

Comment:
perpend all values of options with "TrendLineType." this way, it will simply OnChartTrendlineChanged()